lib/gcra/redis_store.rb

```diff
@@ -13,19 +15,23 @@ class RedisStore
   return 1
   EOF
     CAS_SCRIPT_MISSING_KEY_RESPONSE = 'key does not exist'.freeze
+    SCRIPT_NOT_IN_CACHE_RESPOSNE = 'NOSCRIPT No matching script. Please use EVAL.'
```

Comment:
You have a typo on `RESPOSNE` -> `RESPONSE`, so `SCRIPT_NOT_IN_CACHE_RESPONSE `

Reply:
Thank you! Fixed in 3b1ea2f4f80843fd3a52ed7949c9a8fdfc848e43.